### DISCUSSION TITLE
Github Student Pack Renewal

### DISCUSSION THREAD

**Discussion by KazimFedxD** [community/community#135133] [GitHub Education]
### Select Topic Area

Question

### GitHub Feature Area

GitHub Education

### Body

i am going to college now and my github education pack is bout to expire if i renew it with my college acc will i receive all the benifits again (also the one i have claimed) or does it stay as is.

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @KazimFedxD ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by KazimFedxD** [reply]
cant you just answer it?
